src/components/Label/Label.js

```diff
@@ -116,6 +116,7 @@ export const Labeling = injector(observer(({
               name="dataview"
               minWidth={200}
               showResizerLine={false}
+              type={'label'}
```

Comment:
It's very unclear from such name what it can be, just in this exact place in the code.
Maybe "labeling" at least? Or even "quickview"?